nerdlandbot/helpers/TranslationHelper.py

```diff
@@ -13,3 +13,12 @@ async def get_culture_from_context(ctx: commands.Context) -> str:
     guild = await get_guild_data(guild_id)
     return guild.culture
 
+
+async def get_culture_from_id(guild_id: int) -> str:
```

Comment:
this function basically wraps lines 13 and 14 above so maybe best to replace the lines there with a call to this one using `guild_id` to avoid repetition of the code. better for future maintainability